apps/hash-graph/libs/graph/src/store/fetcher.rs

```diff
@@ -510,9 +522,15 @@ where
                     actor_id,
                     authorization_api,
                     temporal_client,
-                    fetched_ontology_types.data_types,
-                    ConflictBehavior::Skip,
-                    DATA_TYPE_RELATIONSHIPS,
+                    fetched_ontology_types
+                        .data_types
+                        .into_iter()
+                        .map(|(data_type, metadata)| CreateDataTypeParams {
+                            schema: data_type,
+                            classification: metadata.classification,
+                            relationships: DATA_TYPE_RELATIONSHIPS,
+                            conflict_behavior: ConflictBehavior::Fail,
```

Comment:
why the change from `Skip` to `Fail`?

Reply:
Did this by accident, thanks for spotting!